app/Controllers/userController.php

```diff
@@ -110,12 +135,7 @@ public function manageAction() {
 
 		Minz_View::prependTitle(_t('admin.user.title') . ' · ');
 
-		// Get the correct current user.
-		$username = Minz_Request::param('u', Minz_Session::param('currentUser'));
-		if (!FreshRSS_UserDAO::exist($username)) {
-			$username = Minz_Session::param('currentUser');
-		}
-		$this->view->current_user = $username;
+		$this->view->current_user = Minz_Request::param('u');
```

Comment:
When simply browsing to `?c=user&a=manage`, `$this->view->current_user` seems to be null, and I get the following error:
```
<br />
<b>Fatal error</b>:  Uncaught Error: Call to a member function execute() on boolean in ./FreshRSS/app/Models/EntryDAO.php:879
Stack trace:
#0 ./FreshRSS/app/Controllers/userController.php(142): FreshRSS_EntryDAO-&gt;count()
#1 ./FreshRSS/lib/Minz/Dispatcher.php(118): FreshRSS_user_Controller-&gt;manageAction()
#2 ./FreshRSS/lib/Minz/Dispatcher.php(47): Minz_Dispatcher-&gt;launchAction('manageAction')
#3 ./FreshRSS/lib/Minz/FrontController.php(84): Minz_Dispatcher-&gt;run()
#4 ./FreshRSS/p/i/index.php(46): Minz_FrontController-&gt;run()
#5 {main}
  thrown in <b>./FreshRSS/app/Models/EntryDAO.php</b> on line <b>879</b><br />
```

Reply:
Things seem to work fine with the previous code, for this little section

Reply:
@Alkarex That's weird. I cannot reproduce.
My error_reporting is set to E_ALL. My user doesn't have authentication. I cannot think of something else that could break that.
Any ideas ?

Reply:
When landing on the page the first time, `Minz_Request::param('u')` is not defined, is it?

Reply:
It's not defined but set automatically to ```false```.
By the way, I am using mysql. Maybe you're using something different.

Reply:
Tested with SQLite, not tested yet with MySQL - but it would be an error if it does not fail :-P
Anyway, `$entryDAO->count()` (as well as `$databaseDAO->size()` probably) should not be called with an undefined (or false) username.
So I suggest to either keep the previous code, which loaded the default user (seems like the easiest), or move the count elsewhere.

Reply:
@Alkarex I've kept what I was doing previously but I've added a validation on the user. This way, we don't have an error.
I've tried on Sqlite but I couldn't make frss work :'(